jaxrs-api/src/main/java/javax/ws/rs/core/Response.java

```diff
@@ -1296,6 +1311,10 @@ public default Status toEnum() {
          * 307 Temporary Redirect, see {@link <a href="http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.3.8">HTTP/1.1 documentation</a>}.
          */
         TEMPORARY_REDIRECT(307, "Temporary Redirect"),
+        /**
+         * 308 Temporary Redirect, see {@link <a href="https://tools.ietf.org/html/rfc7538">HTTP/1.1 documentation</a>}.
```

Comment:
The link text should say something like `The Hypertext Transfer Protocol Status Code 308 (Permanent Redirect)`, see RFC 7538

Reply:
Actually this is not @mithunsasidharan's fault: That style was chosen by Oracle years before, as you can see in the already existing status codes. But I share your opinion, we should not continue that bad style.

Reply:
I was only referring to the link text. I suggest that it reads `308 Temporary Redirect, see {@link <a href="https://tools.ietf.org/html/rfc7538">The Hypertext Transfer Protocol Status Code 308 (Permanent Redirect)</a>}.` or similar

Reply:
@mkarg @sdaschner : Its the same style we chose so going by that.. I can refactor all in another PR once we decide on it.

Reply:
It's not about the style, I'm just commenting on the link text; that particular RFC 7538 is called "The Hypertext Transfer Protocol Status Code 308 (Permanent Redirect)".

Reply:
@sdaschner : Can you check now.. I've fixed that !